grpccache.go

```diff
@@ -190,7 +190,7 @@ func NoCache(ctx context.Context) context.Context {
 	return context.WithValue(ctx, noCacheKey, struct{}{})
 }
 
-func getNoCache(ctx context.Context) bool {
+func GetNoCache(ctx context.Context) bool {
```

Comment:
The Go linter would say "this exported func should be documented or unexported."